### ISSUE TITLE
Pods Update 2.8.20 Breaks Conditional Display with Beaver Builder 

### ISSUE THREAD

**Issue by 1DayWebs** [pods-framework/pods#6537]
Roll back to 2.8.17 solves problem - I have not checked versions between 2.8.17 and 2.8.20 to confirm.

When updating to 2.8.20 rows that are set to conditional display only if a field is set are appearing as visible even when the field is not set.

**Comment by sc0ttkclark** [comment]
Closing as fixed as part of 2.8.21 since we haven't heard back. Verified with the BB team a few months ago.

**Comment by pen1111** [comment]
I found a discussion about the problem I'm facing. I'm using Beaver Builder and Pods 2.9.13, and the conditional display for Beaver Builder elements is not working.

Previously, I set up the logic for displaying elements based on the presence of data in custom fields. That is, if data is set in the field, the block is displayed, and if there is no data, the block is hidden.

I see that people are writing that a similar issue has been fixed, but unfortunately, it is still relevant.
